scripts/popup.js

```diff
@@ -0,0 +1,142 @@
+        <i class="" id="workDetailClose"><img src="./images/work-detail-cancel.png" alt="icon-cancel"></i>
+        </div>
+        <!-- description -->
+        <div class="frame2">
+            <span class="name">${projects[id - 1].tags[0]}</span>
+            <span class="dot"></span>
+            <span class="role">${projects[id - 1].tags[1]}</span>
+            <span class="dot"></span>
+            <span class="year">${projects[id - 1].tags[2]}</span>
+        </div>
+        <img src="${projects[id - 1].img}" alt="${projects[id - 1].title}" class="workDetailImg">
+        <div class="workDetailInstructions">
+        <p class="workDescription workDetailDescription">
+           ${projects[id - 1].detail}
+        </p>
+        <!-- categories -->
+        <div>
+        <ul class="worksCategories">
+            <li>${projects[id - 1].categories[0]}</li>
+            <li>${projects[id - 1].categories[1]}</li>
+            <li>${projects[id - 1].categories[2]}</li>
+        </ul>
+        <hr>
+        <div class="workDetailFooter">
+        <button id="actionbtn" class="worksBtn worksDetailBtn worksDetailBtn " href="${projects[id - 1].liveVersion}">See live
+        <i class="" id=""><img src="./images/workDetailIcon1.png" alt="workDetailIcon1"></i>
+      </button>
+      <button id="actionbtn" class="worksBtn worksDetailBtn worksDetailBtn" href="${projects[id - 1].source}">See source
+      <i class="" id=""><img src="./images/workDetailIcon2.png" alt="workDetailIcon2" class="workDetailIcon2"></i>
+      </button>
+      </div>
+      </div>
+      </div>
+   </div>
+    <!-- end body -->
+ </div>`;
+  wrapper.style.display = 'block';
+  popup.innerHTML = html;
+  popup.style.display = 'block';
+  const workDetailClose = document.getElementById('workDetailClose');
+  workDetailClose.addEventListener('click', () => {
+    wrapper.style.display = 'none';
+    popup.style.display = 'none';
+  });
+}
+projects.forEach((project) => {
+  const html = `<div class="worksContainer " id="work${project.id}">
+   
+   <img src="${project.img}" alt="${project.title}" class="worksImg">
+   <!-- body -->
+   <div class="worksBody">
+       <h1 class="worksBodytitle">${project.title}</h1>
+       <!-- description -->
+       <div class="frame2">
+           <span class="name">${project.tags[0]}</span>
+           <span class="dot"></span>
+           <span class="role">${project.tags[1]}</span>
+           <span class="dot"></span>
+           <span class="year">${project.tags[2]}</span>
+       </div>
+       <p class="workDescription">
+          ${project.description}
+       </p>
+       <!-- categories -->
+       <ul class="worksCategories">
+           <li>${project.categories[0]}</li>
+           <li>${project.categories[1]}</li>
+           <li>${project.categories[2]}</li>
+       </ul>
+       <button class="worksBtn" id="${project.id}">See Project</button>
+  </div>
+   <!-- end body -->
+</div>`;
```

Comment:
<img width="766" alt="1" src="https://user-images.githubusercontent.com/77843050/218097050-a81b992a-7e24-494f-8bea-83f0e9c88a68.png">

- [ ] On the mobile version, I found this issue in the design, you should do better fixing the style before you jump on the next milestone
- [ ] Kindly arrange the paragraph font-family as per the one used in the correct design